### DISCUSSION TITLE
JSON output; Adding additional text key

### DISCUSSION THREAD

**Discussion by pngoo** [Purfview/whisper-standalone-win#219] [Ideas]
Hello! Apologies if this is not where this post is supposed to go but I was curious if it was possible, when outputting json, to include an additional key 'text' outside of all the segments listed to include the aggregation of all the text in each segments just like the result from the original whisper output?

**Comment by Purfview** [comment]
Why you need it?
You can get just text with `--output_format=text`.

**Reply by pngoo** [reply]
Ah it was just a quality of life thing when i output the json format for what i'm doing instead of having to output just text i like the statistics associated with the words and segments. I was also curious if you had plans for the --initial_prompt option to take in a txt file rather than typing it out in the CLI?

**Reply by Purfview** [reply]
> I was also curious if you had plans for the --initial_prompt option to take in a txt file rather than typing it out in the CLI?

Do you have code for this? Or is this implemented anywhere?

**Reply by pngoo** [reply]
Unfortunately I don't have code for this. I was hoping that I may potentially be able to get the source files (I did not see any when downloading-only the executable) to branch off and make edits myself to enable capability such as this and other nuances that i would like to add. Again apologies if I'm making any dumb suggestions/statements.

**Reply by Purfview** [reply]
CLI is almost same as there -> https://github.com/openai/whisper/blob/main/whisper/transcribe.py

Make it work there, then I can add it here.

**Comment by Purfview** [comment]
Implemented in r189.1
